### ISSUE TITLE
"Recently Added" not working after updating to LibreELEC 7.0.1

### ISSUE THREAD

**Issue by d0cx** [hippojay/plugin.video.plexbmc#172]
Hi. I decided to update OpenELEC v6.0.3 to LibreELEC v7.0.1 and everything I've tried have worked without issues, except for one thing. When selecting "Recently Added", it no longer sorts the file list as it did on OpenELEC v6.0.3. When I checked the current "Sort by: " option on OpenELEC v6.0.3 on a Recently Added file list, it displayed "Default", while on LibreELEC v7.0.1, it has no such option, and it seems to default to "Play count". This is probably not an issue pertaining this plugin but it would be nice to have the "Recently Added" default sorting option restored. I'll continue looking over to see if I can find a way to fix it.

**Comment by d0cx** [comment]
Seems like someone commented the same on the Github that this plugin was forked from... and that person was also using Kodi Jarvis 16.1 (which is the Kodi version on LibreELEC v7.0.1).

https://github.com/HooliganHarls/plugin.video.plexbmc/issues/7

**Comment by d0cx** [comment]
I did the changes in this pull request https://github.com/hippojay/plugin.video.plexbmc/pull/160/files and it seems to do the trick. The Recently Added now works as expected.

**Comment by molle83** [comment]
Hi!
I'am new with raspberry pi and plexbmc.
I newly installed plexbmc 4.0 beta 4 on my raspberry pi running kodi 16 with xbian and i also have problems with sorting with "recently added"
I found a solution to solve the problem here https://github.com/hippojay/plugin.video.plexbmc/pull/160/files but for me it didnt work, proberly i have done something wrong when changed the lines in the py-files "plexsignin.py" and plexbmc.py.

[plex_signin.txt](https://github.com/hippojay/plugin.video.plexbmc/files/404466/plex_signin.txt)
[plexbmc.txt](https://github.com/hippojay/plugin.video.plexbmc/files/404465/plexbmc.txt)


I found this files in \XBIAN\xbian-kodi\addons\plugin.video.plexbmc-4.0.0beta4\resources\lib and not in a "plex"-folder.
Another thing is that i dont the folder "lib" with the file named "init.py" in resources/lib/ndg/.

Is thete a new version of plexbmc where this problem is resolved or can some one help me with the py-files?
